deploy/template.yaml

```diff
@@ -79,31 +79,31 @@ Mappings:
       DNSSUFFIX: review-c.dev.account.gov.uk
       CLIENTS: '[{"jwksEndpoint":"https://ipvstub.review-c.dev.account.gov.uk/.well-known/jwks.json","clientId":"5C584572","redirectUri":"https://ipvstub.review-c.dev.account.gov.uk/redirect"}]'
       MESSAGERETENTIONPERIODDAYS: 345600 # Default: 4 days
-      AUTHSESSIONTTL: "604800" # 7 days in seconds
+      AUTHSESSIONTTL: "950400" # 11 days in seconds
```

Comment:
please update the TTL value in https://govukverify.atlassian.net/wiki/spaces/FTFCRI/pages/3592978468/Kiwi+Default+Session+TTLs

Reply:
Done :)